scripts/launch_aws_cluster.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set +x
+
+bash <(curl -s https://raw.githubusercontent.com/dcos/dcos/master/test_util/terraform_init.sh)
+
+./terraform init
```

Comment:
If I understand this line correctly, this is already done here:

https://github.com/dcos/dcos/blob/09608a1c0e0bf2237ffd0afb1111faa3414a6abb/test_util/terraform_init.sh#L79